usaspending_api/common/csv_stream_s3_to_pg.py

```diff
@@ -94,6 +96,50 @@ def _stream_and_copy(
         raise exc
 
 
+def _download_and_copy(
```

Comment:
Where are unit and integration tests for this new function?

Reply:
Still need to get this in